backend/.env.example

```
@@ -1,3 +1,7 @@
+DEBUG=True
+DJANGO_ENV=development
+SECRET_KEY=change-me-to-a-secure-key-at-least-32-bytes-long
+ALLOWED_HOSTS=localhost,127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_HOSTS key should go before the DEBUG key

(UnorderedKey)